### ISSUE TITLE
Expo Image - PagerView flicker bug

### ISSUE THREAD

**Issue by henrymoulton** [expo/expo#23830]
### Minimal reproducible example

https://github.com/henrymoulton/expo-image-pagerview-bug

### Summary

When using Expo Image and PagerView together it waits for the image to load/render despite PagerView not being lazy. FastImage doesn't have this behaviour.

See screen recording: https://www.loom.com/share/e2c1fce71f494cb3a6d6f9cee7bbd990

### Environment

```
➜  expo-image-pager-view-bug git:(main) npx expo-env-info

  expo-env-info 1.0.5 environment info:
    System:
      OS: macOS 13.3.1
      Shell: 5.9 - /bin/zsh
    Binaries:
      Node: 16.20.0 - ~/.nvm/versions/node/v16.20.0/bin/node
      Yarn: 1.22.17 - /opt/homebrew/bin/yarn
      npm: 8.19.4 - ~/.nvm/versions/node/v16.20.0/bin/npm
      Watchman: 2023.02.06.00 - /opt/homebrew/bin/watchman
    Managers:
      CocoaPods: 1.11.3 - /Users/henrymoulton/.rbenv/shims/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 22.4, iOS 16.4, macOS 13.3, tvOS 16.4, watchOS 9.4
    IDEs:
      Android Studio: 2022.1 AI-221.6008.13.2211.9619390
      Xcode: 14.3.1/14E300c - /usr/bin/xcodebuild
    npmPackages:
      expo: ~49.0.5 => 49.0.6 
      react: 18.2.0 => 18.2.0 
      react-native: 0.72.3 => 0.72.3 
    npmGlobalPackages:
      eas-cli: 3.15.0
      expo-cli: 6.3.10
    Expo Workflow: managed
```

**Comment by henrymoulton** [comment]
Hey @tsapeta would love to bin FastImage and this is blocking us at the moment, let me know if there's anything I can do - should I file this with the PagerView team?

**Comment by tsapeta** [comment]
Hi @henrymoulton, I've made a PR #25343 that hopefully fixes that issue 😉 Sorry for keeping you waiting.

**Comment by tsapeta** [comment]
It's been published in `expo-image@1.8.0` 🙂

**Comment by henrymoulton** [comment]
Legend 🥳 thanks @tsapeta!
